server/portal/apps/webhooks/models.py

```diff
@@ -1,9 +1,6 @@
-from __future__ import unicode_literals
-
 from django.db import models
 from django.conf import settings
 from datetime import datetime
-from portal.utils.fields import JSONField
```

Comment:
do we need to drop the jsonfield-related workarounds in server/portal/utils/fields.py?

Reply:
Not until the squashed migrations are used, then we can delete these util files as well as the old migrations that reference them.
https://github.com/TACC/Core-Portal/pull/798#discussion_r1187621518

Reply:
👍